apps/web/app/(app)/environments/[environmentId]/settings/members/components/AddMemberModal.tsx

```diff
@@ -1,18 +1,11 @@
 "use client";
 
 import { Modal } from "@formbricks/ui/Modal";
-import {
-  Select,
-  SelectContent,
-  SelectGroup,
-  SelectItem,
-  SelectTrigger,
-  SelectValue,
-} from "@formbricks/ui/Select";
 import { Button } from "@formbricks/ui/Button";
 import { Input } from "@formbricks/ui/Input";
 import { Label } from "@formbricks/ui/Label";
-import { useForm, Controller } from "react-hook-form";
+import { useForm } from "react-hook-form";
+import AddMemberRole from "@formbricks/ee/RoleSwitch/components/AddMemberRole";
```

Comment:
as we have nearly all of our components as module imports instead of default imports please also import this as module import :-)